docs/get-started-with-datahub.md

```diff
@@ -0,0 +1,62 @@
+# Get Started With DataHub
+
+This section is focused on helping you get up and running with DataHub as quickly as possible.
+
+## Get Started with Self-Hosted DataHub
+
+This is a great place to start if you're new to DataHub and are interested in taking the open-source project for a spin.
+
+You'll learn how to:
+
+* Deploy DataHub on your local machine
+* Ingest sample data to get a feel for what is possible in DataHub
+* Add and invite users to DataHub
+* Troubleshoot common issues
+
+### Prerequisites
+
+Before you go further, ensure you have the following installed:
+
+* [Docker](https://docs.docker.com/get-docker/)
+* [jq](https://stedolan.github.io/jq/download/)
+* [Docker Compose](https://github.com/docker/compose/blob/master/INSTALL.md) - if using Linux
+
+:::note
+
+Make sure to allocate enough hardware resources for Docker! 
+We recommend the following configuration: 2 CPUs, 8GB RAM, 2GB Swap area, and 10GB disk space.
+
+:::
+
+<p>
+<a
+    className='button button--primary button--lg'
+    href="quickstart">
+    Get Started with Self-Hosted DataHub
+</a>
+</p>
+
+## Get Started with Managed DataHub
+
+[Acryl Data](https://www.acryldata.io/product) provides a fully managed, premium version of DataHub.
+
+In this section, you'll learn how to:
+
+* Add and invite users to DataHub
+* Configure Slack notifications
+* Understand Approval Workflows
+
+### Prerequisites
+
+Before you go further, you'll need to have a DataHub instance provisioned. The Acryl Integrations Team will provide you with the following once it has been deployed:
+
+* The URL for your Acryl instance, i.e. `https://<your-domain-name>.acryl.io`
+* Admin account credentials for logging into the DataHub UI
+
+<p>
+<a
+    className='button button--primary button--lg'
+    href="authentication/guides/add-users">
```

Comment:
This hyperlinks out to the first article in the Managed DataHub section; we'll need to update the link if we change the ordering of articles in the sidebar.